CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [19.1.2] - 2022-03-23
```

Comment:
We tend to just put changes into the `[Unreleased]` section right up until the point where we actually make the release, since we might merge multiple PRs and it will get more awkward to merge this file if they're modifying the version headers.

Reply:
done